Assets/Scripts/OrchestraArmy/Room/Room.cs

```diff
@@ -313,5 +380,14 @@ int NumberOfFloors()
             return count;
         }
 
+        public void OnEvent(EnemyDeathEvent invokedEvent)
+        {
+            _numberOfEnemies--; //one enemy less
+            if(_numberOfEnemies<1) //if all enemies are dead
```

Comment:
Is wel heel erg op elkaar geplakt, zorg dat alle classes goed geformatteerd zijn!